Haddock build failure with version 0.20.0.0

src/Language/Haskell/LSP/Core.hs:425:15: error:
        parse error on input ‘-- $/ notifications should/could be ignored by server.
                  -- Don't log errors in that case.
                  -- See https://microsoft.github.io/language-server-protocol/specifications/specification-current/#-notifications-and-requests.’
        |
    425 |               -- $/ notifications should/could be ignored by server.
        |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...

--- a/ChangeLog.md
+++ b/ChangeLog.md
@@ -1,5 +1,9 @@
 # Revision history for haskell-lsp
 
+## 0.20.0.1
+
+* Fix Haddock generation syntax error
+
 ## 0.20.0.0
 
 * Don't log errors for '$/' notifications (@jinwoo)

--- a/haskell-lsp.cabal
+++ b/haskell-lsp.cabal
@@ -1,5 +1,5 @@
 name:                haskell-lsp
-version:             0.20.0.0
+version:             0.20.0.1
 synopsis:            Haskell library for the Microsoft Language Server Protocol
 
 description:         An implementation of the types, and basic message server to

--- a/src/Language/Haskell/LSP/Core.hs
+++ b/src/Language/Haskell/LSP/Core.hs
@@ -422,7 +422,7 @@ hh mVfs wrapper mh tvarDat json = do
           case mh of
             Just h -> h req
             Nothing
-              -- $/ notifications should/could be ignored by server.
+              -- '$/' notifications should/could be ignored by server.
               -- Don't log errors in that case.
               -- See https://microsoft.github.io/language-server-protocol/specifications/specification-current/#-notifications-and-requests.
               | isOptionalNotification req' -> return ()
@@ -1031,12 +1031,12 @@ flushDiagnosticsBySource tvarDat maxDiagnosticCount msource = join $ atomically 
         resSendResponse ctx $ NotPublishDiagnostics
           $ J.NotificationMessage "2.0" J.TextDocumentPublishDiagnostics params
 
--- |=====================================================================
+-- =====================================================================
 --
 --  utility
 
 
--- |
+-- 
 --  Logger
 --
 setupLogger :: Maybe FilePath -> [String] -> Priority -> IO ()

